test/CMakeLists.txt

```diff
@@ -43,3 +43,32 @@ compile_benchmark_test(cxx03_test)
 set_target_properties(cxx03_test
     PROPERTIES COMPILE_FLAGS "${CXX03_FLAGS}")
 add_test(cxx03 cxx03_test --benchmark_min_time=0.01)
+
+# Add the coverage command(s)
+if (${CMAKE_BUILD_TYPE} MATCHES "[Cc]overage")
+  find_program(GCOV gcov)
+  find_program(LCOV lcov)
+  find_program(GENHTML genhtml)
+  # TODO: support more compilers and generators
+  find_program(MAKE make)
```

Comment:
i'm not sure you want to restrict this at all.. is there no way to get the 'make' command from cmake?